build/depvers.sh

```diff
@@ -56,9 +58,6 @@ for (( i=0; i < ${#pkginfo[@]}; i+=2 )); do
   if test "${git}" -eq 1; then
     vers=$(git -C "${dir}" rev-parse HEAD 2>/dev/null )
   else
-    # file ownership sometimes doesn't work in docker and hg gets grumpy about
-    # this file, which is basically empty anyway, so just delete it
-    rm "${dir}/.hg/hgrc"
```

Comment:
If you're going to remove the hg support, fully remove it. I think that's a mistake, though. This script worked fine with hg dependencies. The fact that `go get -u` doesn't is no reason to partially cripple this script.